src/plays/product-entry-form/components/AddProduct/ProductEntryForm.js

```diff
@@ -0,0 +1,136 @@
+import { useState } from "react";
+import ErrorModal from "../UI/ErrorModal";
+
+export default function ProductEntryForm(props) {
+	const [inputName, setInputName] = useState("");
```

Comment:
Why didn;t you think of handling the state as object than many states?

Reply:
I never tried using object for managing the state, it was from my beginner lessons which I learned, but surely will do that.